### ISSUE TITLE
support for IIIF images 

### ISSUE THREAD

**Issue by dancorpse** [lovasoa/dezoomify#175]
http://www.harvardartmuseums.org/collections/object/228046?position=0# , despite that the image is downloadable, I'd like to try dezoomify,  the image url is https://ids.lib.harvard.edu/ids/iiif/43183651/info.json, right? But it's not workable in dezoomify , I don't know what's wrong.

**Comment by tubage170** [comment]
harvardartmuseums.org you can use online request.

**Comment by dancorpse** [comment]
Do you mean the image can be downloaded? Oh, I 'd just like to find out what's the zoom image url. :)

**Comment by tubage170** [comment]
yes,harvardartmuseums.org will send you by email. if you use online request.

**Comment by AlexanderDeLargeBrotherSir** [comment]
<img width="2550" height="2075" alt="Image" src="https://github.com/user-attachments/assets/12e3358f-4444-4e98-98b1-dc6b076dd21e" />

https://images.harvardartmuseums.org/urn-3:HUAM:770505

@dancorpse, close this please.
